package.json

```diff
@@ -38,19 +38,19 @@
     "@babel/cli": "~7.7.0",
     "@babel/core": "~7.7.0",
     "@babel/plugin-proposal-class-properties": "~7.7.0",
-    "@babel/plugin-proposal-object-rest-spread": "~7.6.0",
-    "@babel/plugin-transform-runtime": "~7.6.0",
+    "@babel/plugin-proposal-object-rest-spread": "~7.7.0",
```

Comment:
`7.7.0` doesn't exist, neither does `7.6.0`

Reply:
🤔 
https://www.npmjs.com/package/@babel/plugin-proposal-object-rest-spread

Reply:
WAIT. I totally get what you're saying.